components/Dashboard/Modals/ProfileDialog/index.js

```diff
@@ -8,9 +8,12 @@ import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
 import Hidden from '@material-ui/core/Hidden';
-import SettingsIcon from '@material-ui/icons/Settings';
 import { makeStyles } from '@material-ui/core/styles';
 
```

Comment:
please keep imports consistent. We shouldn't have spaces between imports from external packages. Reference the other pages to see what I mean.